File: HelloWorld/app/src/main/java/com/example/helloworld/MainActivity.java
package com.example.helloworld;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    private static final String LOG_TAG = "MainActivity";
    private int mCount = 0;
    private TextView mShowText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mShowText = findViewById(R.id.show_count);
        Log.d(LOG_TAG, "Hello World");

    }

    public void showToast(View view){
        Toast toast = Toast.makeText(this, R.string.toast_message, Toast.LENGTH_SHORT);
        toast.show();
    }

    public void countUp(View view){
        mCount++;
        if (mShowText != null){
            mShowText.setText(Integer.toString(mCount));
        }
    }
}

File: RoutineCheck/app/src/main/java/com/example/routinecheck/MainActivity.kt
package com.example.routinecheck

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val timing = ArrayList<String>()
        addRoutineTime(timing)
        val adapterTime = ArrayAdapter<String>(this, android.R.layout.simple_list_item_1, timing)
        routine_frequency.adapter = adapterTime

        btn_routine_save.setOnClickListener {
            var routineName = routine_name.text
            var routinedescrp = routine_descrip.text
            var routineTime = routine_frequency.selectedItem

            Toast.makeText(this, "Your selection is $routineName $routineTime", Toast.LENGTH_LONG).show()
        }

    }

    private fun addRoutineTime(timing: ArrayList<String>) {
        timing.add("Hourly")
        timing.add("Daily")
        timing.add("Weekly")
        timing.add("Monthly")
        timing.add("Yearly")
    }
}
